src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
```

Comment:
let's remove the empty lines after a class declaration as per our [common mistakes](https://github.com/mate-academy/jv-oop/blob/master/checklist.md)
same for other classes

Reply:
thx, fixed.